convert2rhel/utils.py

```diff
@@ -501,21 +501,35 @@ def download_pkg(
     if ret_code != 0:
         loggerinst.warning("Output from the yumdownloader call:\n%s" % (output))
 
-        if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:
-            loggerinst.critical(
-                "Couldn't download the %s package. This means we will not be able to do a"
-                " complete rollback and may put the system in a broken state.\n"
-                "Check to make sure that the %s repositories are enabled"
-                " and the package is updated to its latest version.\n"
-                "If you would rather ignore this check set the environment variable"
-                " 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'." % (pkg, system_info.name)
-            )
+        # add in the new envar that should be changed to along with the old one
+        allow_old_envar_names = ("CONVERT2RHEL_INCOMPLETE_ROLLBACK", "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK")
+
+        # search for either envar in the system to see if it has been set
+        if any(envvar in os.environ for envvar in allow_old_envar_names):
+
+            # check to see which envar was used
+            if "CONVERT2RHEL_INCOMPLETE_ROLLBACK" not in os.environ:
+                # give the warning about the old envar not being used anymore if it is uesed
+                loggerinst.warning(
+                    "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'"
+                    " environment variable.  Please switch to 'CONVERT2RHEL_INCOMPLETE_ROLLBACK' instead."
+                )
+
+                # give regular warning if the new envar is used
+                loggerinst.warning(
+                    "Couldn't download the %s package. This means we will not be able to do a"
+                    " complete rollback and may put the system in a broken state.\n"
+                    "'CONVERT2RHEL_INCOMPLETE_ROLLBACK' environment variable detected, continuing conversion." % (pkg)
+                )
```

Comment:
@Andrew-ang9 that warning message should be indented to be outside of that upper if statement, like this: 

```suggestion
            # check to see which envar was used
            if "CONVERT2RHEL_INCOMPLETE_ROLLBACK" not in os.environ:
                # give the warning about the old envar not being used anymore if it is uesed
                loggerinst.warning(
                    "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'"
                    " environment variable.  Please switch to 'CONVERT2RHEL_INCOMPLETE_ROLLBACK' instead."
                )

            # give regular warning if either the new or old envvar is used
            loggerinst.warning(
                "Couldn't download the %s package. This means we will not be able to do a"
                " complete rollback and may put the system in a broken state.\n"
                "'CONVERT2RHEL_INCOMPLETE_ROLLBACK' environment variable detected, continuing conversion." % (pkg)
            )
```

Reply:
I have done this and will push it soon to the PR! Thank you for pointing this out!